packaging_automation/dbconfig.py

```diff
@@ -4,7 +4,7 @@
 from attr import dataclass
 from sqlalchemy import Column, INTEGER, TIMESTAMP, TEXT
 from sqlalchemy import create_engine
-from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import declarative_base
```

Comment:
Why did we need to do this change?

I am curious especially because we did not bump sqlalchemy version here.

Reply:
There was a warning as below
/home/runner/work/tools/tools/packaging_automation/dbconfig.py:34: MovedIn20Warning: The ``declarative_base()`` function is now available as sqlalchemy.orm.declarative_base(). (deprecated since: 2.0) (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)